components/Companies.tsx

```diff
@@ -2,13 +2,51 @@ import { companies } from "../data/companies";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { faLocationDot } from "@fortawesome/free-solid-svg-icons";
 import DataCard from "./DataCard";
+import Search from "./Search";
+import { useState } from "react";
 
 export default function Companies() {
+  // SEARCH HANDLER //
+  const [search, setSearch] = useState("");
+  const [select, setSelect] = useState("");
+  const handleSearch = (e) => {
+    setSearch(e.target.value);
+  };
+  const handleSubmit = (e) => {
```

Comment:
We can probably remove the submit button and the Handle Submit function since it sorts as you type now